feat: pg17 compatibility (#5)
CrunchyData/pgmonitor-extension: CHANGELOG.md

2.0.0
=====

FEATURES
--------
 - Compatible with PostgreSQL 17

BREAKING CHANGES
----------------

>  - PG17 restructured the pg_stat_bgwriter catalog information. Extension has restructured background writer/checkpointer stats around those changes.

Reviewer: ```suggestion
 - PG17 restructured the pg_stat_bgwriter catalog information. This extension has been restructured around those changes:
```
?
